web/handelsregister/handelsregister/urls.py

```diff
@@ -149,18 +149,27 @@ class Search(cls):
 }
 
 
-@api_view()
-@renderer_classes(
-    [OpenAPIRenderer, renderers.CoreJSONRenderer])
-def hr_schema_view(request):
-    generator = schemas.SchemaGenerator(title='Handelsregister API')
-    return response.Response(generator.get_schema(request=request))
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Bouwdossiers API",
```

Comment:
Dit is wel handelsregister. Niet Bouwdossiers

Reply:
Ouch. Gefixed.